src/app/compliance-rule-configuration/configure-compliance-rule/configure-compliance-rule.component.html

```diff
@@ -4,8 +4,8 @@ <h1 mat-dialog-title>Configure Compliance Rule</h1>
     <button mat-button (click)="fillInTestData()">Create Test Data</button>
 
     <h4>Compliance Rule {{this.utils.getId(data)}}</h4>
-    <h5>Description:</h5>
-    <p>{{data.description}}</p>
+    <h5>Issue Type:</h5>
+    <input matInput type="text" value="{{this.issueType}}">
     <div *ngIf="this.complianceRuleParameters.length > 0">
```

Comment:
somehow this is not getting populated on my end

Reply:
I think I know why: you are only showing existing assignments, not the parameters. Therefore, even if there are parameters, they are not shown if they haven't been assigned a value yet.